### ISSUE TITLE
Chem Entity causes increase of Gene that causes target Disease - why is this returned for MVP1 - what drugs may treat?

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#139]
Steps:

1. CI UI, What drugs may treat dentin dysplasia:  https://ui.ci.transltr.io/results?q=72a5d7c4-c88c-4fb9-936c-b2a89f885b66
2. Expand the fourth result (scored 98.1) 

![image](https://user-images.githubusercontent.com/38321826/220993331-02a1d293-689d-453d-b7ab-06244eb07d31.png)

3.  in ARAX UI it looks like its coming from BTE:  https://arax.ncats.io/?r=72a5d7c4-c88c-4fb9-936c-b2a89f885b66
![image](https://user-images.githubusercontent.com/38321826/220993632-c653ce0b-81aa-4e40-8722-e096228e2c1c.png)

**Comment by andrewsu** [comment]
Apologies, knowing nothing about Anti-Mullerian Hormone or dentin dysplasia, why is this a problematic answer?

**Comment by sierra-moxon** [comment]
I swear I had added a screenshot to this while doing the triage because I had the same question.  Ah well, here it is. :)   I came to the conclusion that it might be because the qualifiers indicate that this chemical increases the abundance of the gene, and the gene has a "causes" relationship with the disease.  As a naive user, if a gene (or gene product) causes a disease, then more of the gene (or gene product) would be the opposite desired effect of a chemical treatment?   I couldn't tell from the "causes" edge metadata if "causes" was from an abundance or deficit of the gene (or gene product).  

<img width="654" alt="Screen Shot 2023-02-23 at 12 28 51 PM" src="https://user-images.githubusercontent.com/4561292/221022865-4f1b66aa-aa7f-459e-a1d4-c73932420279.png">

**Comment by andrewsu** [comment]
Thanks @sierra-moxon.  Yes, that makes sense.  But I think because, as you noted, the nature of the relationship between the gene and the disease isn't specified more concretely, so I think this is a result that we'd still want to show. (And it's actually a design principle in BTE that directionality is a secondary consideration. i.e., I'd much rather show an answer that reflects a biologically correct relationship in the "wrong" direction than not show the relationship at all.  Directionality can be adjusted in many cases, and that's something I'd rather the user / SME evaluate anyway.)

Unless there is another concern here that I'm missing, I suggest closing this issue.

**Comment by sierra-moxon** [comment]
the only other two things I could find were that: 

ACVR1 activity/expression - causes -> dental dysplasia was assessed in mice (Acvr1) (https://pubmed.ncbi.nlm.nih.gov/30519900/ - SEMMEDDB) 
- [found curated annotations in MGI](https://www.informatics.jax.org/allele/allgenoviews/MGI:3689350), with many more details about the genotype that might prevent curators at MGI from declaring Acvr1 causative?  (not sure, SME expertise would be very valuable here). 

MIS - affects (increases abundance of) -> ACVR1  was assessed in rats (https://pubmed.ncbi.nlm.nih.gov/16687449/ - SEMMEDDB) 
- [found curated annotations for this paper in RGD, but they don't reference ACVR1](https://rgd.mcw.edu/rgdweb/report/reference/main.html?id=2303145#objectsAnnotated)

**Comment by sstemann** [comment]
@sierra-moxon does this change anything?  in the same result set i see both increase and decrease:

![image](https://user-images.githubusercontent.com/38321826/221215962-6092ea69-02a7-4d54-bda8-8f4a3cb57147.png)

**Comment by sierra-moxon** [comment]
I did reach out to Monarch Initiative for clarification here, and we found that there are assertions in the SRI Reference KG between Acvr1 and "abnormal mandible morphology" ([from a different pub](https://www.alliancegenome.org/gene/MGI:87911#phenotypes)).  We don't see the "dental dysplasia" assertion directly in a curated resource, but in my opinion, SEMMEDDB is adding value here ahead of curation. 

Some things that would have helped this investigation go more quickly and result in more confidence:
- "knock out" designation on the gene would be helpful
- "taxon" metadata would be helpful 
- Link the gene itself to the appropriate external resource to tell me more about its phenotypes (this is already in discussion via other tickets in this repo: #91) 
- Be able to see all the publications in Translator that have Acvr1 "mouth" phenotype assertions (something like a link to "related search results" or "did you mean?" functionality)

This is probably a separate set of findings from those in Sarah's original post, which she has clarified here.  I'll put this in "Review at TAQA" for now?  :)
